### ISSUE TITLE
Flatpak Vesktop does not autostart with sytem

### ISSUE THREAD

**Issue by Plarpoon** [Vencord/Vesktop#273]
**Host details:** KDE 5.28.8 with kernel 6.5.0-14 Kubuntu.

_**Flatpak installation type.**_

**Vesktop details**:
Stable 250836 (84e28a2)
Vencord 6573c47 (Vesktop V0.4.4)
Electron 27.1.2
Chromium 118.0.5993.144
Linux 64-Bit

**Relevant settings:** "Start with System" set on enabled in the "Vesktop Settings" left menu tab.

Despite such an option being enabled the process does not automatically start at system boot.

**Comment by Plarpoon** [comment]
**Found workaround:**

Adding Vesktop to the autostart list here in System Settings
![image](https://github.com/Vencord/Vesktop/assets/40299833/207e8bd7-42d7-4c19-b2d7-338adc938a40)

**Comment by Plarpoon** [comment]
I invite others with different DE or WM to test it out and report under here if possible. All my machines run KDE at the moment so I can test it only with for the time being.

**Comment by Vendicated** [comment]
yeah this is a known issue. the way vesktop autostart currently works is that it just creates a .desktop file in ~/.config/autostart

that functions for normal installs, but not for flatpaks. the fix is to instead use the [feeedesktop autostart portal](https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.Background.html)

related: https://github.com/flathub/flathub/pull/4749#issuecomment-1837225178

**Comment by Plarpoon** [comment]
Oh that's great, I am dumb I even read that yesterday then forgot about it today. Forgive me!

If you don't mind I would keep the issue open as it contains a workaround for people who might need it, but you are free to close it if you want to! And thanks for all the effort putting into the project!

**Comment by Plarpoon** [comment]
I suppose this issue should follow #242
